backend/src/main/java/ca/gov/dtsstn/cdcp/api/web/v1/model/mapper/SubscriptionModelMapper.java

```diff
@@ -25,6 +25,9 @@ public abstract class SubscriptionModelMapper extends AbstractModelMapper {
 
 	@Nullable
 	public CollectionModel<SubscriptionModel> toModel(String userId, @Nullable Iterable<Subscription> subscriptions) {
+		if (subscriptions == null || subscriptions.spliterator() == null) {
```

Comment:
`subscriptions.spliterator()` will never be null here

```suggestion
		if (subscriptions == null) { return CollectionModel.empty(); }
```

Reply:
Superfluous check removed in commit [9fbe7e7](https://github.com/DTS-STN/canadian-dental-care-plan/pull/1865/commits/9fbe7e70e363ba40945580d5841081219b4bf781).